pkgs/top-level/python-packages.nix

```
@@ -3243,7 +3243,9 @@ self: super: with self; {
 
   dnachisel = callPackage ../development/python-modules/dnachisel { };
 
-  dnf4 = callPackage ../development/python-modules/dnf4 { };
+  dnf-plugins-core = callPackage ../development/python-modules/dnf-plugins-core { };
+
+  dnf4-unwrapped = callPackage ../development/python-modules/dnf4 { };
```

```suggestion
  dnf4 = callPackage ../development/python-modules/dnf4 { };
```
we don't need to rename this